integration_tests/src/main/python/hive_delimited_text_test.py

```diff
@@ -301,6 +317,7 @@ def test_hive_text_default_enabled(spark_tmp_path, data_gen, spark_tmp_table_fac
         lambda spark: read_hive_text_table(spark, table_name),
         conf={})
 
+
```

Comment:
nit extra line

Reply:
Would it be alright if we kept 2 lines between functions? Without it, I see a Flake8 warning:
https://www.flake8rules.com/rules/E302.html

Reply:
ah sorry.. I keep forgetting this is python. Never mind